src/components/Tabs/Tabs.scss

```diff
@@ -328,11 +328,10 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
 
-    &:focus .Title::before {
-      background: var(--p-border-hovered);
-    }
-
     &:active .Title::before {
+      background: var(--p-action-primary-pressed);
```

Comment:
I made the color to be `--p-action-primary-pressed` so it matched the format PRE new design language as you can see in this gif:

![preDesignTab](https://user-images.githubusercontent.com/43223543/100690757-3d61fa00-3355-11eb-9316-d74586e1810c.gif)